doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -0,0 +1,398 @@
+    "\n",
+    "For the demo, we'll directly load a pretrained ResNet model from `torchvision.models` and construct a {class}`~ray.train.torch.TorchCheckpoint` which includes the preprocessor. You can also load your own Ray AIR checkpoint from your previous Train/Tune experiments. You can find more details about checkpoint loading at the [AIR `Checkpoint` API reference](air-checkpoint-ref)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchvision import models\n",
+    "from ray.train.batch_predictor import BatchPredictor\n",
+    "from ray.train.torch import TorchCheckpoint, TorchPredictor\n",
+    "\n",
+    "# Load the pretrained resnet model and construct a checkpoint\n",
+    "model = models.resnet152(pretrained=True)\n",
+    "checkpoint = TorchCheckpoint.from_model(model=model, preprocessor=preprocessor)\n",
+    "\n",
+    "# Build a BatchPredictor from checkpoint\n",
+    "batch_predictor = BatchPredictor(checkpoint, TorchPredictor)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, run batch prediction using {meth}`BatchPredictor.predict(...) <ray.train.batch_predictor.BatchPredictor.predict>`!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-02-14 18:47:07,508\tINFO batch_predictor.py:184 -- `num_gpus_per_worker` is set for `BatchPreditor`.Automatically enabling GPU prediction for this predictor. To disable set `use_gpu` to `False` in `BatchPredictor.predict`.\n",
+      "Map_Batches: 100%|██████████| 50/50 [00:03<00:00, 13.88it/s]\n",
+      "Map Progress (2 actors 1 pending): 100%|██████████| 9/9 [00:14<00:00,  1.62s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "predictions = batch_predictor.predict(\n",
+    "    ds,\n",
+    "    feature_columns=[\"image\"],\n",
+    "    keep_columns=[\"label\"],\n",
+    "    batch_size=128,\n",
+    "    max_scoring_workers=3,\n",
+    "    num_gpus_per_worker=1,\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We specified several parameters in `predict()`:\n",
+    "- `feature_columns` specifies which columns are required for the model. The selected columns will be concatenated to build a batch tensor.\n",
```

Comment:
I don't think the part about concatenation is true. if multiple columns are provided, then it will just be fed as a dictionary of torch tensors to the model.

for concatenation, you need to use the Concatenator preprocessor.

but since in this example we are only using 1 column, we don't need to mention concatenation at all.

Reply:
Ok. I have removed the sentence about feature concatenation.